bap-vibes/src/bir_passes.ml

```diff
@@ -655,10 +655,11 @@ module ABI = struct
       ~(tgt : Theory.target)
       ~(sp_align : int)
       ~(hvars : Hvar.t list)
-      ~(entry_blk : blk term) : (blk term list * Higher_var.t list) KB.t =
+      ~(entry_blk : blk term)
+    : (blk term list * Higher_var.t list * String.Set.t) KB.t =
```

Comment:
A comment explaining what these three things are would be helpful (or a record time with descriptive names, if you prefer)